packages/react-scripts/fixtures/kitchensink/integration/webpack.test.js

```diff
@@ -17,8 +17,10 @@ describe('Integration', () => {
         doc.getElementsByTagName('style')[0].textContent.replace(/\s/g, '')
       ).to.match(/html\{/);
       expect(
-        doc.getElementsByTagName('style')[1].textContent.replace(/\s/g, '')
+        doc.getElementsByTagName('style')[0].textContent.replace(/\s/g, '')
```

Comment:
i just realize that this only test the `style-loader` behaviour in test and dev mode, need to revert this and find out how to test the behaviour for `npm run build` output